test/Message.test.cpp

```diff
@@ -1,4 +1,5 @@
 #include <cstring>
+#include <sstream>
```

Comment:
Is this include necessary ?

Reply:
Yes. It is necessary to be able to compile with the #define private public below.